### ISSUE TITLE
Creating repo with README.md fails

### ISSUE THREAD

**Issue by flixh** [yuki-kimoto/gitprep#77]
When creating a new repo and letting gitprep initialize it with a README.md it fails with an "Internal error". The production log says:
```
[Tue Jun  2 08:48:53 2015] [error] /_new: Can't execute git commit: /usr/bin/git --git-dir=/tmp/yUw48uYnBl/work --work-tree=/tmp/yUw48uYnBl/work commit -q --author=USER <USER@localhost> -m first commit at /home/gitprep/gitprep/extlib/lib/perl5/DBIx/Connector.pm line 315.
```

I'm pretty sure this used to work and creating the repo without README still works.
Any ideas?

Felix

**Comment by yuki-kimoto** [comment]
You should set name and email at first.

    git config --global user.name "kimoto"
    git config --global user.email "kimoto@example.com"

**Comment by flixh** [comment]
Unfortunately this is not the reason here, since user.name and user.email are set for the gitprep user:

gitprep@machine:~$ git config -l
user.name=gitprep
user.email=root@machine
gitprep@maschine:~$

Regards,
Felix

**Comment by flixh** [comment]
A lot of debugging later, everything is back to working as expected. But I'm still unsure whether I have identified the root cause:
Instead of the git commit failure reported above, at some point (no idea why) the errors reported changed to something related to permissions "...cannot chdir to /root...". Which was caused by starting gitprep with
```
sudo -u gitprep /home/gitprep/gitprep/gitprep
```
within /root and gitprep trying chdir to this directory at some point.

I now wrote a basic Debian init.d script that first changes to /home/gitprep and then starts gitprep. With that setup everything is working again.

Felix
